package.json

```diff
@@ -184,7 +184,8 @@
     "build": "node ./scripts/build/build.js",
     "build:website": "node ./scripts/build-website.js",
     "gen:changelog": "node ./scripts/generate-changelog.js",
-    "debug": "node bin/prettier.js --ignore-path=.prettierignore"
+    "debug": "node bin/prettier.js --ignore-path=.prettierignore",
+    "postinstall": "node -e \"try{fs.existsSync('.vscode/settings.json')||fs.writeFileSync('.vscode/settings.json',fs.readFileSync('.vscode/settings.example.json'))}catch(_){}\""
```

Comment:
Turns out we can't use an extra script file to copy the file, because this will run when `yarn install prettier@prettier/prettier`

Reply:
Is there any reason we don't use `prepare`?

Reply:
yarn>=2 don't run `prepare`.

Reply:
I am surprised, which means that the `Prepare` in the Babel warehouse did not work as expected.
But when we use Yarn 4, maybe we can use it to complete this.
https://yarnpkg.com/configuration/yarnrc#enableConstraintschecks

Reply:
I don't understand how `enableConstraintschecks` can do this. But yarn plugin should be able to do it.

Reply:
When `EnableConstraintschecks` is true, Yarn runs every time` yarn.config.cjs`.
https://github.com/babel/babel/blob/main/yarn.config.cjs

Reply:
I'm going to use yarn plugin instead.